Libraries/Hotcakes.Commerce/BusinessRules/OrderTasks/StartPaypalExpressCheckout.cs

```diff
@@ -193,38 +201,26 @@ public override bool ProcessCheckout(OrderTaskContext context)
 
                         Orders.OrderNote note = new Orders.OrderNote();
                         note.IsPublic = false;
-                        note.Note = "Paypal Order Accepted With Paypal Order Number: " + expressResponse.Token;
+                        note.Note = "Paypal Order Accepted With Paypal Order Number: " + expressResponse.Result<Order>().Id;
                         context.Order.Notes.Add(note);
                         if (context.HccApp.OrderServices.Orders.Update(context.Order))
                         {
 							string urlTemplate;
                             if (string.Compare(context.HccApp.CurrentStore.Settings.PayPal.Mode, "Live", true) == 0)
                             {
-								urlTemplate = "https://www.paypal.com/cgi-bin/webscr?cmd=_express-checkout&token={0}";
+								urlTemplate = "https://www.paypal.com/checkoutnow?token={0}";
```

Comment:
Please make static strings like this a `constant`.

Reply:
OK done it.